output/oxygen_level/original_geneID.txt

```diff
@@ -0,0 +1 @@
+""
```

Comment:
empty file?

Reply:
The "original_geneID.txt" and "latent_geneID.txt" are both empty since no genes were found that were better modeled by the transformation in gene space (original) compared to in latent space and vice versa.  

I will remove these files and have changed the script ("predictive_genes.py") to only write to file if the DataFrame is not empty.